public/app/types/teams.ts

```diff
@@ -15,6 +15,7 @@ export interface TeamDTO {
 // This is the team resource with permissions and metadata expanded
 export interface Team {
   id: number; // TODO switch to UUID
+  uid: string; // Prefer UUID
```

Comment:
Is it really a UUID or a grafana short ID?

Reply:
Grafana short UID. can't really call it UUID :sweat: